src/extensions/styles/style.scss

```diff
@@ -54,7 +54,7 @@ body.coil-missing-id:not( .coil-no-gating ),
 body.monetization-not-initialized.coil-gate-tagged-blocks:not( .coil-no-gating ) {
 
 	*.coil-hide-monetize-users {
-		display: none;
+		display: block;
```

Comment:
@SarahCoilAccount is there a reason we're changing a class labelled "hide" to display block?